langchain/llms/textgen.py

```diff
@@ -162,7 +165,10 @@ def _get_parameters(self, stop: Optional[List[str]] = None) -> Dict[str, Any]:
         if self.stopping_strings and stop is not None:
             raise ValueError("`stop` found in both the input and default params.")
 
-        params = self._default_params
+        if self.preset is None:
+            params = self._default_params
+        else:
+            params = {"preset": self.preset}
```

Comment:
shouldnt this still use self._default params?

Reply:
it will use the preset defined in textgen, no need for additional params. all this params are defined in the preset.

Reply:
@hwchase17 is all the checks must pass? when it will be merged?